conf_collectd_plugin.py

```diff
@@ -0,0 +1,375 @@
+import sys
+import re
+
+import install_utils as utils
+import config
+
+COLLECTD_HOME = "/etc/collectd"
+COLLECTD_CONF_DIR = COLLECTD_HOME+"/managed_config"
```

Comment:
pep8 did not flag any issue for this file.  I understand that operator
needs space around it as I have caught a few.  I believe the pep8 program
may not detect all errors.  I will read through the pep8 doc that you sent.

On Wed, Jul 6, 2016 at 1:05 PM, Jason Bau <notifications@github.com> wrote:

> In conf_collectd_plugin.py
> <https://github.com/wavefrontHQ/install/pull/22#discussion_r69801719>:
>
> > @@ -0,0 +1,375 @@
> > +import sys
> > +import re
> > +
> > +import install_utils as utils
> > +import config
> > +
> > +COLLECTD_HOME = "/etc/collectd"
> > +COLLECTD_CONF_DIR = COLLECTD_HOME+"/managed_config"
>
> yeah, it should flag stylistic issues that you should fix.
>
> FYI, this is what pep8 is: https://www.python.org/dev/peps/pep-0008/
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/wavefrontHQ/install/pull/22/files/cd4dd95748662ae81a055276b6fdc466da95710a#r69801719>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ALZQ30rxNC6SSrTjZ-0cRat2BhzfPOQQks5qTApugaJpZM4JFrB2>
> .
>